app/models/user.rb

```diff
@@ -63,6 +63,17 @@ def pbs
         .union_all(runs.by_category(nil))
   end
 
+  def non_pbs
+    ids = []
+    categories = []
+    pbs.map do |run|
+      ids << run.id
+      categories << run.category_id
+    end
+
+    runs.where(category_id: categories).where.not(id: ids)
```

Comment:
Long shot, but any simple way to select non-PBs using just SQL?

Reply:
Do you mean to remove the harvest steps before this line?